app/shared/modals/reservation-info/ReservationInfo.js

```diff
@@ -108,6 +104,13 @@ export default function ReservationInfoModal(props) {
           name="Varauksen viestit"
           reservationId={reservation.id}
         />
+        <hr />
+        <Row>
+          <Col className="details-row reservation-participants" xs={6}>
```

Comment:
- [ ] This should take the whole modal width as in the mockup, instead of half of it.